testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/UIUtils.java

```diff
@@ -133,6 +133,10 @@ public static String getTextInputValue(WebElement input) {
     }
 
     public static void setTextInputValue(WebElement input, String value) {
+        if (StringUtils.isEmpty(getTextFromElement(input)) && StringUtils.isEmpty(value)) {
```

Comment:
Can you explain these changes?

Reply:
Of course I can, it always adds a "1" in a field and then removes it even if the field is empty, that means that angular will send a empty string to the backend instead of omitting the value. This is why bugs like this could slipped through.

Reply:
Strictly speaking the admin console shouldn't send an empty string, which we need to address in the new admin console.

Not sure it is a good idea to add this to all fields that are set, as it would then behave differently to how a user uses the console. I would rather think we should add some specific tests for these cases.

The bug here could have been detected by better testing of the REST APIs though, so don't need testing in the admin console really.